src/styles/theming/common/switch/_sap_horizon.scss

```diff
@@ -0,0 +1,74 @@
+:root {
```

Comment:
i like this approach but we need to unify the chosen solution over the project. if we're going with separate files for each component - let's do it for all components. if we're placing all the variables in one file - let's do this for all components.

let's raise that question on the call so we can discuss it and agree with the team.

Reply:
> i like this approach but we need to unify the chosen solution over the project. if we're going with separate files for each component - let's do it for all components. if we're placing all the variables in one file - let's do this for all components.
> 
> let's raise that question on the call so we can discuss it and agree with the team.

I chose this approach because amount of changed variables is enormous. It would be unreadable in case of storing such variables in one file per theme. Another thing is that for all fiori variables are the same, thus, copy-pasting this variables into each theme file doesn't make much sense.

Reply:
You can't just choose approach without discussing it with the team. There are 30+ components in this library. If you do it for this one you need to update the rest. Fundamental-styles will not become the mess fundamental-ngx is, where each dev is influencing their way of coding. And I totally +1 what Platon says, it's not about the approach, it's about the consistency. If we do it for one, we do it for all components.

Reply:
Discussed, agreed.